website/docs/setup/guides/unit-testing.md

```diff
@@ -203,5 +203,15 @@ The test passed! Now if we configure our CI system to execute our test we can
 ensure that uncle Cecil remains in Shoreditch after any future config change.
 What an insufferable hipster he is.
 
+import Alert from '@site/src/components/Alert';
```

Comment:
I like that you added this, but I actually don't think it's warranted for this guide. It seems a bit out of place here. We can mention this in the docs for these fields via https://github.com/timberio/vector/issues/1790.